src/components/atoms/charts/Calendar3D/index.jsx

```diff
@@ -0,0 +1,377 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+//> Additional packages
+// Used to display the time in a readable format
+import moment from "moment";
+
+//> CSS
+import "./calendar3d.scss";
+//#endregion
+
+//#region > Components
+/**
+ * @class A three dimensional calendar which displays each days contributions,
+ *        and contribution related statistics e.g. busiest days and streaks.
+ */
+class Calendar3D extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.myInput = React.createRef();
```

Comment:
Add comment like in `Calendar2D`.
```suggestion
    // Create reference to HTML canvas
    this.myInput = React.createRef();
```